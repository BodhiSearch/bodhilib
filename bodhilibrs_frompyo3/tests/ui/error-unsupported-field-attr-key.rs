use frompyo3::FromPyO3;

#[derive(FromPyO3)]
struct TestUnsupportedFieldAttrKey {
  #[frompyo3(unknown = "metadata")]
  path: String,
}

fn main() {}
