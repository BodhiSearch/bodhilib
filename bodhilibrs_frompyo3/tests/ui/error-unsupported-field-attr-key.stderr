error: Unsupported field level attribute: 'unknown'
 --> tests/ui/error-unsupported-field-attr-key.rs:5:14
  |
5 |   #[frompyo3(unknown = "metadata")]
  |              ^^^^^^^
